Foundation/NSPropertyList.swift

```diff
@@ -113,6 +113,8 @@ internal func _expensivePropertyListConversion(input : AnyObject) -> Any {
         return data
     } else if let number = input as? NSNumber {
         return number
+    } else if let url = input as? NSURL {
```

Comment:
Hm, I'm pretty sure `NSBundle.infoDictionary` included a value somewhere that was an NSURL — triggering a fatalError without it... What should I do instead?